src/main/java/blue/endless/glow/model/gltf/impl/GLTFData.java

```diff
@@ -32,234 +36,384 @@ public class GLTFData {
 	public static final int GL_USHORT = 5123;
 	public static final int GL_INT    = 5125;
 	public static final int GL_FLOAT  = 5126;
-	
+	//Wrap
+	public static final int GL_REPEAT = 10497;
+
+	public String[] extensionsUsed = new String[0];
+	public String[] extensionsRequired = new String[0];
+	public GLTFAccessor[] accessors = new GLTFAccessor[0];
+	public GLTFAnimation[] animations = new GLTFAnimation[0];
 	public GLTFAsset asset = new GLTFAsset();
-	public GLTFBufferView[] bufferViews = new GLTFBufferView[0];
 	public GLTFBuffer[] buffers = new GLTFBuffer[0];
```

Comment:
I don't hate reordering the GTLF components here to be alphabetical, but it sure does make for a messy diff. Next time let's confine that to a formatting PR.